include/lsst/afw/math/Kernel.h

```diff
@@ -129,9 +129,9 @@ using boost::serialization::make_nvp;
 
     public:
         typedef double Pixel;
-        typedef PTR(Kernel) Ptr;
-        typedef CONST_PTR(Kernel) ConstPtr;
-        typedef PTR(lsst::afw::math::Function2<double>) SpatialFunctionPtr;
+        typedef std::shared_ptr<Kernel> Ptr;
+        typedef std::shared_ptr<Kernel const> ConstPtr;
```

Comment:
I'm confused. Where are we keeping and where are we removing `Ptr` member types? At first I thought we would remove `PTR` and `CONST_PTR` but keep the rest, but looking at the other PRs it seems like a bit of a mixed bag.